addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -546,14 +547,13 @@ def l10n_in_grouping_key_generator(base_line, tax_values):
                         line_code = "state_cess"
                 else:
                     for gst in ["cgst", "sgst", "igst"]:
+                        if any(tag in tags for tag in self.env.ref("l10n_in.tax_tag_%s_rc"%(gst))):
+                            is_reverse_charge = True
                         if any(tag in tags for tag in self.env.ref("l10n_in.tax_tag_%s"%(gst))):
                             line_code = gst
             return {
                 "tax": tax,
```

Comment:
Is this key still used somewhere ?
Apparently you do not use it anymore here:
https://github.com/odoo/odoo/pull/112877/files#diff-db9aa0435189465d0f97f085e1933656530760810c45daedf29306fb9177312fL575

Reply:
@rigr-odoo We don't get you on this.
we are using here https://github.com/odoo/odoo/pull/112877/files#diff-db9aa0435189465d0f97f085e1933656530760810c45daedf29306fb9177312fL575

Reply:
You are using the key `is_reverse_charge`, but not the key `tax` if I'm not mistaken so there is no point in returning it in the dictionary ?

Reply:
we need to know whether this invoice has a reverse charge tax or not.
so what do you suggest to do it?

Reply:
Returning this:
```py
return {
    "is_reverse_charge": is_reverse_charge,
    "line_code": line_code,
}
```
instead of this:

```py
return {
     "tax": tax,
     "is_reverse_charge": is_reverse_charge,
     "line_code": line_code,
}
```
because here https://github.com/odoo/odoo/pull/112877/files#diff-db9aa0435189465d0f97f085e1933656530760810c45daedf29306fb9177312fL575-R575
you use `is_reverse_charge` but not `tax`

Reply:
@rigr-odoo okay, got it.
@niai-odoo make sure, that is not used in any other modules.
if not found then remove it.

Reply:
@jva-odoo 
It is used here : https://github.com/odoo/odoo/pull/112877/files#diff-db9aa0435189465d0f97f085e1933656530760810c45daedf29306fb9177312fL577

Reply:
@niai-odoo still we can remove it by sending `tax_amount` but it's a conflict with the real amount so we not remove `tax`
@rigr-odoo it's okay for you?

Reply:
Yep sure